__tests__/app.test.js

```diff
@@ -33,4 +34,62 @@ describe("API Testing", () => {
       });
     });
   });
+  describe("/api/reviews", () => {
+    describe.only("GET requests", () => {
+      test("Should return a status code of 200", () => {
+        return request(app).get("/api/reviews").expect(200);
+      });
+      test("Should return an object containing an array of reviews", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((res) => {
+            res.body.reviews.forEach((obj) => {
+              expect(obj).toMatchObject({
+                review_id: expect.any(Number),
+                title: expect.any(String),
+                category: expect.any(String),
+                designer: expect.any(String),
+                owner: expect.any(String),
+                review_body: expect.any(String),
+                review_img_url: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+              });
+            });
+          });
+      });
+      test("Should return an array of objects containing a comment_count property", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((res) => {
+            res.body.reviews.forEach((obj) => {
+              expect(obj).toEqual(
+                expect.objectContaining({
+                  comment_count: expect.any(Number),
+                })
+              );
+            });
+          });
+      });
+      test("Should return an array of objects sorted by date descending", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((res) => {
+            expect(res.body.reviews[0]).toEqual(
```

Comment:
Have a look at jest-sorted here to be able to test this more effectively as right now your only testing the dats of the first and last ones, not all of them :)